static/js/compose.js

```diff
@@ -713,7 +713,28 @@ function get_or_set(fieldname, keep_leading_whitespace) {
     };
 }
 
-exports.stream_name     = get_or_set('stream');
+function get_contents(fieldname, keep_leading_whitespace) {
+
+    return function () {
```

Comment:
I think we can further kill off this pattern by something like this:

~~~
function trim_field_val(selector) {
   return $.trim($(selector).val());
}
function stream_name_get () {
   return trim_field_val("stream");
}